pkg/v1/webhooks/certificates.go

```diff
@@ -63,50 +63,64 @@ func addCertsToWebhookConfigs(ctx context.Context, k8sclient kubernetes.Interfac
 	return nil
 }
 
-func NewTLSSecret(ctx context.Context, secretName, serviceName, certPath, keyPath, namespace string) (*corev1.Secret, error) {
-	secret, err := resources.MakeSecretInternal(ctx, secretName, namespace, serviceName)
-	if err != nil {
-		return nil, err
-	}
+// func NewTLSSecret(ctx context.Context, secretName, serviceName, certPath, keyPath, namespace string) (*corev1.Secret, error) {
+//	secret, err := resources.MakeSecret(ctx, secretName, namespace, serviceName)
+//	if err != nil {
+//		return nil, err
+//	}
+
+//	return secret, nil
+//}
 
```

Comment:
Could you add godoc comments to all these exported functions please? I'm not sure why the linter is not catching it.

Reply:
done!